neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -150,6 +161,11 @@ type VirtualMachineSpec struct {
 	// propagated to the VM.
 	// +optional
 	TargetRevision *RevisionWithTime `json:"targetRevision,omitempty"`
+
+	// Enable network monitoring on the VM
+	// +kubebuilder:default:=false
+	// +optional
+	EnableNetworkMonitoring *bool `json:"enableNetworkMonitoring,omitempty"`
```

Comment:
idle thought: If we're concerned about mismatched iptables rules, maybe instead of a boolean, this should explicitly give the rules to track (and, potentially a mapping for how they should be exposed?)

Not sure, don't know enough about iptables to say either way -- just putting it out there as an idea.

Reply:
This would increase complexity of the code (and tie it to the cloud/ repo) without any benefits - we'd still need to query iptables, so just querying all the rules seems simpler. The mismatch issue was not the rules being wrong, just the rules being absent.